draft-ietf-quic-multipath.md

```diff
@@ -633,7 +640,7 @@ for a given Path ID to its peer. Endpoints SHOULD NOT introduce discontinuity
 in the issuing of Path IDs through their connection ID advertisements as path initiation
 requires available connection IDs for the same Path ID on both sides. For instance,
 if the maximum Path ID limit is 2 and the endpoint wants to provide connection IDs
-for only one Path ID inside range [1, 2], it should select Path ID 1 (and not Path
+for only one Path ID inside range (1, 2), it should select Path ID 1 (and not Path
```

Comment:
The original

Reply:
The problem is kramdown. The angle brackets are confused with a library reference, and we cannot compile the text. It might be possible to escape kramdown by using something like "\[1, 2\]", but I have not tried that.

Reply:
Tried it now, it seems to work.